src/messagesStore/actions.js

```diff
@@ -0,0 +1,23 @@
+export function requestMessages() {
+  return { type: 'REQUEST_MESSAGES' };
+}
+
+export function receiveMessages(messages) {
+  return {
+    type: 'RECEIVE_MESSAGES',
+    messages,
+  };
+}
+export function setMessageFieldText(text) {
+  return {
+    type: 'SET_MESSAGE_FIELD_TEXT',
+    text,
+  };
+}
+export function submitMessage(userIndex, text) {
+  return {
+    type: 'SUBMIT_MESSAGE',
+    userIndex,
+    text,
+  };
```

Comment:
`text` isn't used in the reducer. Can go.

```suggestion
export function submitMessage(userIndex) {
  return {
    type: 'SUBMIT_MESSAGE',
    userIndex,
  };
```